### PR TITLE
Order of edits and dependency adds was swapped

### PR DESCRIPTION
**Pull request by dsblank**
If you follow the directions, you'll get a compile error because ICommandPalette isn't used yet.

### CODE REVIEW

File: docs/source/developer/extension_tutorial.rst [hunk 1]
```diff
@@ -257,7 +248,16 @@ will pass an instance of ``ICommandPalette`` as the second parameter of
 in that function. The second ``console.log`` line exists only so that
 you can immediately check that your changes work.
 
-Run the following to rebuild your extension.
+Now you will need to install these dependencies. Run the following commands in the
+repository root folder install the dependency and save it to your
```
Comment (jasongrout) [inline docs/source/developer/extension_tutorial.rst]:
```suggestion
repository root folder to install the dependencies and save them to your
```

Comment (jasongrout) [reply docs/source/developer/extension_tutorial.rst]: (might as well fix the typo in the line as well)

**Comment by jasongrout** [comment]
Thanks! I suggested an edit to fix the preexisting typo in the line as well.

**Comment by dsblank** [comment]
Thanks @jasongrout !

**Pull request merged**